test/unit/tree-key-cache-preOrderBreadthFirstSearch.spec.ts

```diff
@@ -184,6 +185,11 @@ describe(TreeKeyCache.name, () => {
 			map.set('a1', JSON.stringify('a'));
 			map.set('a1:b1:c1', JSON.stringify('b'));
 			map.set('a:b:c2', JSON.stringify('v2'));
+			target['internal'] = new TreeInternalControl(
+				target['options'],
+				target,
+				target['storage'],
+			);
```

Comment:
internal is defined during construction. To not crate a new Target, I've just set a new TreeInternalControl with the desired options updated